modules/apps/portal-search/portal-search-web/src/main/java/com/liferay/portal/search/web/internal/sort/display/context/builder/SortDisplayContextBuilder.java

```diff
@@ -59,21 +59,20 @@ public SortDisplayContext build() {
 		List<SortTermDisplayContext> sortTermDisplayContexts =
 			_buildSortTermDisplayContexts();
 
-		List<DropdownItem> actionDropdownItems = _buildActionDropdownItems();
-
 		sortDisplayContext.setAnySelected(
 			isAnySelected(sortTermDisplayContexts));
-		sortDisplayContext.setSelectedSortTermDisplayContext(
-			getSelectedSortTermDisplayContext(sortTermDisplayContexts));
-
-		sortDisplayContext.setActionDropdownItems(actionDropdownItems);
 
 		sortDisplayContext.setDisplayStyleGroupId(getDisplayStyleGroupId());
 		sortDisplayContext.setParameterName(_parameterName);
 		sortDisplayContext.setParameterValue(getParameterValue());
 		sortDisplayContext.setRenderNothing(isRenderNothing());
 		sortDisplayContext.setSortPortletInstanceConfiguration(
 			_sortPortletInstanceConfiguration);
+
+		sortDisplayContext.setActionDropdownItems(
+			_getActionDropdownItems(sortTermDisplayContexts));
+		sortDisplayContext.setSelectedSortTermDisplayContext(
+			getSelectedSortTermDisplayContext(sortTermDisplayContexts));
 		sortDisplayContext.setSortTermDisplayContexts(sortTermDisplayContexts);
```

Comment:
Can these be alphabetized by the method?

Reply:
Along with `setAnySelected`. It doesn't seem like there's any reason for the line breaks.

Reply:
That SF rule is strange in this case. I don't see anything used afterward other than returning `sortDisplayContext` which is all of these lines together are modifying.

Reply:
The order of the `set*` matters? That's very bug-prone. I'm not seeing the connection to the other set methods. 🤔

Reply:
ok! just pushed the changes as we said. SF still told me `There should be an empty line before line "67", as we finished referencing variable "sortTermDisplayContexts"` so I added a line break, but at least the order doesn't matter and it's alphabetical.